packages/sdk/src/wallet-utils/validate-tx.js

```diff
@@ -0,0 +1,19 @@
+import {
+  encodeTransactionPayload as encodeInsideMessage,
+  encodeTransactionEnvelope as encodeOutsideMessage,
+} from "../encode/encode.js"
+import {invariant} from "@onflow/util-invariant"
+import {TRANSACTION} from "../interaction/interaction"
+
+const validateSignableTransaction = signable => {
+  invariant(
+    signable.interaction.tag === TRANSACTION,
+    "Signable payload must be transaction"
+  )
+
+  return signable.roles.payer
+    ? encodeOutsideMessage(signable.voucher) === signable.message
+    : encodeInsideMessage(signable.voucher) === signable.message
+}
+
+export default validateSignableTransaction
```

Comment:
We generally stick with names exports (instead of default exports), as it helps with tree shaking.

Reply:
Got it. 

*I did export named from [wallet-utils/index.js](https://github.com/onflow/flow-js-sdk/blob/153c04febcf7af982420926d23dd89f2f4a0248d/packages/sdk/src/wallet-utils/index.js)
For importing a la 
`import  {validateSignableTransaction, otherWalletUtil} from "../wallet-utils"`

Let me know if this works or if it's a pattern you'd prefer to avoid :)

Reply:
Updated to named export per convention